planetary-social/nos
# Track mentions autocomplete opened

## Issues covered
#1480 

## Description
This PR tracks mentions when the mentions button is clicked and the user types an "@"

## How to test
1. Build the app.
2. Click on the post button at the bottom of the screen.
3. Type an "@" or press the mentions button.
4. Please check that the following is logged: "Mentions Opened".
